util/log/log.go

```diff
@@ -23,16 +23,20 @@ import (
 	"strconv"
 	"time"
 
-	"github.com/cockroachdb/clog"
 	"golang.org/x/net/context"
 )
 
-const timeFormat = "1/2 15:04:05.000" // m/d h:m:s.nanos
+const humanTimeFormat = "1/2 15:04:05.00" // m/d h:m:s.ss
```

Comment:
I'd rather keep three fractional digits - I can think in milliseconds but centiseconds just look weird.